db/structure.sql

```diff
@@ -945,5 +953,7 @@ INSERT INTO schema_migrations (version) VALUES ('20150619075903');
 
 INSERT INTO schema_migrations (version) VALUES ('20150619090833');
 
+INSERT INTO schema_migrations (version) VALUES ('20150622083615');
```

Comment:
Not a big thing, but this probably wants to be at the end of the file as the migrations should be inserted in order.  Anyone running rake db:migrate will probably end up with things in a different order and we'll have dirty db/structure.sql files.  Probably just need to reset your db before you run the migrations.